src/js/controllers/index.js

```diff
@@ -136,6 +136,10 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     $log.debug('ERROR:', err);
     if (err.code === 'NOTAUTHORIZED') {
       $scope.$emit('Local/NotAuthorized');
+    } else if (err.code === 'NOTFOUND') {
+      $scope.$emit('Local/NotFound');
```

Comment:
Since events' namespace is shared, names should be more descriptive:
Local/NotFound -> Local/BWSNotFound
Local/TimeOut -> Local/BWSTimeOut